Gemfile

```diff
@@ -136,22 +142,31 @@ gem 'tzinfo', '>= 1.2.0'	# required by rails; 1.2.0 has support for *BSD and Sol
 # Windows does not have zoneinfo files, so bundle the tzinfo-data gem.
 gem 'tzinfo-data', platforms: [:mingw, :mswin, :x64_mingw]
 
-# Introduces a scope for Heroku specific gems.
-def on_heroku
-  if ENV['ON_HEROKU'] ||
-     ENV['HEROKU_POSTGRESQL_ROSE_URL'] ||
-     ENV['HEROKU_POSTGRESQL_GOLD_URL'] ||
-     File.read(File.join(File.dirname(__FILE__), 'Procfile')) =~ /intended for Heroku/
+
+# Introduces a scope for gem loading based on a condition
+def if_true(condition)
```

Comment:
Fist of all I thinks its not the scope of this PR, I just generalized a function that already was used for months.

As far as I understood @knu the benefit of bundler 1.10+ would be the removal of 10 lines from our Gemfile, and I don't know if that would be worth any challenge at all (even if its just `sudo gem update bundler`). We would at least need to check Travis, Docker and add notes to our guides.

@knu you reverted the `install_if` changes in 76be1240128dd30ece34c85c14cd8ef9ad8a09f4, do you remember why?